contrib/gitian-build.py

```diff
@@ -41,7 +41,6 @@ def setup():
     if args.is_bionic and not args.kvm and not args.docker:
         subprocess.check_call(['sudo', 'sed', '-i', 's/lxcbr0/br0/', '/etc/default/lxc-net'])
         print('Reboot is required')
-        exit(0)
```

Comment:
Why remove this line?

Reply:
> Why remove this line?

See [my comment below](https://github.com/bitcoin/bitcoin/pull/15236#issuecomment-456756208).